CMakeLists.txt

```diff
@@ -981,7 +981,16 @@ endif()
 # LuaJIT (Scripting Support)
 # ==========================
 if(FLB_LUAJIT)
-  include(cmake/luajit.cmake)
+  find_package(PkgConfig)
+  pkg_check_modules(LUAJIT luajit>=2.1.0)
+  if(LUAJIT_FOUND)
```

Comment:
Let's make this a configurable option to ensure current behaviour is maintained, i.e. we opt in to finding luajit.

Reply:
Which name should I give to that option?

Reply:
`ENABLE_FIND_SYSTEM_LUAJIT` or similar like option should be possible to use here.

Reply:
```diff
 option(FLB_LUAJIT              "Enable Lua Scripting support" Yes)
+option(FLB_ENABLE_FIND_SYSTEM_LUAJIT "Build Lua Scripting with system provided package" No)
```
Perhaps `FLB_LUAJIT_ENABLE_FIND_SYSTEM` would be better alligned, then it would be:
```diff
 option(FLB_LUAJIT              "Enable Lua Scripting support" Yes)
+option(FLB_LUAJIT_ENABLE_FIND_SYSTEM "Build Lua Scripting with system provided package" No)
```

Btw, could be that the I add the same trick for c-ares, miniz and so on.

Reply:
The second one looks better, "namespaces" make interpretation easier.